deploy-manage/deploy/cloud-enterprise/ece-install-offline-images.md

```diff
@@ -22,10 +22,10 @@ The first table contains the stack versions that shipped with the 3.8 version of
 | docker.elastic.co/cloud-assets/kibana:7.17.27-0 |
 | docker.elastic.co/cloud-assets/apm:7.17.27-0 |
 | docker.elastic.co/cloud-assets/enterprise-search:7.17.27-0 |
-| docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.1 |
-| docker.elastic.co/cloud-release/kibana-cloud:8.17.1 |
-| docker.elastic.co/cloud-release/elastic-agent-cloud:8.17.1 |
-| docker.elastic.co/cloud-release/enterprise-search-cloud:8.17.1 |
+| docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.4 |
```

Comment:
This part here should be replaced with the versions from ECE 4.0:
https://github.com/elastic/cloud/blob/ms-122/scala-services/bootstrap/src/main/resources/reference-versions.conf

8.17.3, 8.18.0, 9.0.0

Reply:
Yeah we could actually update ECE 4 to include 8.17.4. It would mean updating the reference-versions on ms-122 as well.